.github/workflows/ci.yml

```diff
@@ -34,3 +34,19 @@ jobs:
       - uses: ros-tooling/action-ros-ci@v0.2
         with:
           target-ros2-distro: ${{ matrix.ros_distribution }}
+
+  lint:
```

Comment:
Maybe run lint first as a fast-fail given that it only take ~20 sec.

Reply:
They all run in parallel so people will get lint feedback quickly either way. I tend to prefer not failing fast - Github actions are free for open source so the extra resources aren't costing us anything, and it can save a round trip if you know that you have to fix both a lint issue and a build/test issue (rather than fixing a lint issue only to find afterwards that you also have a failing test).